packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1199,6 +1199,9 @@ class SemanticsProperties extends DiagnosticableTree {
   /// It'll be appear in accessibility hierarchy as `resource-id`.
   ///
   /// On iOS, this will set `UIAccessibilityElement.accessibilityIdentifier`.
+  ///
+  /// On web, this will set a `flt-semantics-identifier` attribute on the DOM element
+  /// that corresponds to the semantics node. 
```

Comment:
Removing extra white space:

```suggestion
  /// that corresponds to the semantics node.
```